File: Swift/two-sum-ii-input-array-is-sorted.swift
class Solution {
    func twoSum(_ numbers: [Int], _ target: Int) -> [Int] {
        var dict = [Int: Int]()

        for (i, num) in numbers.enumerated() {
            guard let index = dict[target - num] else {
                dict[num] = i + 1
                continue
            }
            return [index, i + 1]
        }

        fatalError("no valid outputs")
    }
}

File: Python/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        d = dict()

        for i, num in enumerate(numbers):
            if (target - num) in d:
                return [d[target - num], i + 1]
            d[num] = i + 1
